src/test/java/uk/gov/hmcts/reform/et/syaapi/service/ApplicationServiceTest.java

```diff
@@ -349,4 +354,40 @@ void shouldMarkApplicationAsViewed() {
             = argumentCaptor.getValue().getGenericTseApplicationCollection().get(0).getValue().getApplicationState();
         assertThat(actualState).isEqualTo("viewed");
     }
+
+    @Nested
+    class UpdateApplicationState {
+        @Test
+        void inProgressWhenResponseWasRequired() {
```

Comment:
Would it be cleaner to use ParameterizedTest?

Reply:
Good idea